boards/shields/mikroe_weather_click/doc/index.rst

```diff
@@ -0,0 +1,59 @@
+.. _mikroe_weather_click_shield:
+
+MikroElektronika Weather Click
+##############################
+
+Overview
+********
+
+The MikroElektronika `Weather Click`_ features the `BME280`_ integrated
+environmental sensor in a `mikroBUS`_ |trade| form factor. The sensor can
+measure relative humidity, barometric pressure, and ambient temperature.
+
+.. figure:: weather-click.webp
+   :align: center
+   :alt: MikroElektronika Weather Click
+
+   MikroElektronika Weather Click (Credit: MikroElektronika)
+
+Requirements
+************
+
+This shield can only be used with a board that provides a mikroBUS |trade|
+socket and defines a ``mikrobus_i2c`` node label for the mikroBUS |trade| I2C
+interface (see :ref:`shields` for more details).
+
+For more information about the BME280 and the Weather Click, see the following
+documentation:
+
+- `Weather Click`_
+- `Weather Click Schematic`_
+- `BME280`_
+- `BME280 Datasheet`_
+
+Programming
+***********
+
+Set ``-DSHIELD=mikroe_weather_click`` when you invoke ``west build``. For
+example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensor/bme280
+   :board: lpcxpresso55s16
+   :shield: mikroe_weather_click
+   :goals: build
```

Comment:
FYI, I don't have a `lpcxpresso55s16` board to test this, so I haven't actually verified that this exact command will work. (I have tested this shield works on a custom board.)

If anybody has the `lpcxpresso55s16` (or any of the other in-tree boards with built-in mikroBUS) + a Weather Click lying around and can test this real quick, it would be nice to just double check this.